src/modules/uv-pdfheaderpanel-module/css/styles.less

```diff
@@ -7,7 +7,6 @@
         color: @text-secondary-color;
 
         button.btn.imageBtn {
-            font-size: 0;
             padding-left: 2px;
```

Comment:
Noticing the text appearing without the screen reader, can we use `.sr-only();` here too?

Reply:
<img width="625" alt="Screenshot 2020-07-16 at 16 16 27" src="https://user-images.githubusercontent.com/8266711/87689109-ba624d80-c77f-11ea-875d-8acbd653a899.png">
For reference from the Netlify ^

Reply:
This is an inadvertently added bug. The `font-size: 0` is probably a fine if unconventional solution for hiding text.